src/sources/video_card.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Credits: https://stackoverflow.com/a/49267844
+ */
```

Comment:
Please move the comment just above the `getVideoCard` function and change the opening part to `/**`. Example: https://github.com/fingerprintjs/fingerprintjs/blob/b4c5d0e2193355661529ada99c546ae12bba71c3/src/sources/color_gamut.ts. Such way IDEs will attach the comment to the function.